spec/features/users/login_spec.rb

```diff
@@ -62,6 +60,42 @@
       expect(page).to have_content("Welcome Admin, #{@admin.name}!")
     end
 
+    it "cannot login user with incorrect email" do 
+      expect(current_path).to eq("/login")
+
+      fill_in :email, with: "incorrect email"
+      fill_in  :password, with: @user.password
+      click_button "Login"
+
+      expect(current_path).to eq("/login")
+
+      expect(page).to have_content("Incorrect email or password.")
```

Comment:
Ah - after reading the user story again I see that this might be why you didn't use the .errors in the controller - we don't want to see if it was email or password specifically, just raise the error

Reply:
Based on the convo above, I'm not sure how to use .errors for an incorrect email (nil user). If anyone has ideas for how to get around this and use .errors, that would be awesome! (...I guess that is assuming that .errors gives us this generic message asked for in the user story).

Reply:
I get it now!